app/presenters/income_benefits_presenter.rb

```diff
@@ -0,0 +1,31 @@
+require 'laa_crime_schemas'
+
+class IncomeBenefitsPresenter < BasePresenter
+  def initialize(benefits)
```

Comment:
Just wondering if we want to call this income_benefits throughout this file for consistency with the rest of this PR, and consistency with the income_payments PR